File: exercicios em C/q13.cpp
#include <stdio.h>

int main(void){
	int A, B, C;
	printf ("\n\nEste programa determina se tres numeros formam um triangulo, e caso forme um triangulo, classifica-o em isosceles, escaleno ou equilatero. Tambem verifique se e um triangulo retangulo, acutangulo ou obtusangulo.");
	printf ("\n\nDigite o prmeiro valor: ");
	scanf ("%d", &A);
	printf ("\n\nDigite o segundo valor: ");
	scanf ("%d", &B);
	printf ("\n\nDigite o terceiro valor: ");
	scanf ("%d", &C);
	if ((A < B + C) && (B < A + C) && (C < A + B)) {
        printf ("\n\nOs tres numeros formam um triangulo!");
        if ((A == B) && (A == C) && (B == C)) {
            printf ("\n\nTemos um triangulo equilatero!");
        }
        else if ((A == B) || (A == C) || (B == C)) {
            printf ("\n\nTemos um triangulo isoceles!");
        }
        else if ((A != B) && (A != C) && (B != C)) {
            printf ("\n\nTemos um triangulo escaleno!");
        }
        
        if ((A * A == ((B * B) + (C * C))) || (B * B == (( A * A ) + (C * C))) || (C * C == ((A * A) + (B * B)))){
        	printf("\nE os valores digitados formam um triangulo retangulo!");
		}
		else if((A * A > (B * B) + (C * C)) || (B * B > ( A * A ) + (C * C)) || (C * C > (A * A) + (B * B))){
			printf("\nE os valores digitados formam um triangulo obtusangulo!");
		}
		else if((A * A < (B * B) + (C * C)) || (B * B < ( A * A ) + (C * C)) || (C * C < (A * A) + (B * B))){
			printf("\nE os valores digitados formam um triangulo acutangulo");
		}
	}
    else {
        printf("\n\nOs tres numeros nao formam triangulo!");
	}
	printf("\n\nObrigado por utilizar nosso sistema!");
	return 0;
}

File: exercicios em Python/q13.py
import os

print("\nEste programa determina se tres numeros formam um triangulo, e classifica-o!")
A = int(input("\n\nDigigite o valor da primeira aresta: "))
B = int(input("\nagora o valor da segundo aresta: "))
C = int(input("\ne por ultimo o valor da terceiro aresta: "))
if ((A < B + C) & (B < A + C) & (C < A + B)):
    print("\n\nOs três números formam um triângulo!")
        
if ((A == B) & (A == C) & (B == C)):
    print("\nTemos um triângulo equilátero!")
    
elif ((A == B) | (A == C) | (B == C)): 
    print("\nTemos um triângulo isósceles!")
    
elif ((A != B) & (A != C) & (B != C)):
    print("\nTemos um triângulo escaleno!")

if ((A * A == ((B * B) + (C * C))) | (B * B == (( A * A ) + (C * C))) | (C * C == ((A * A) + (B * B)))):
    print("\nE os valores digitados formam um triangulo retangulo!")

elif((A * A > (B * B) + (C * C)) | (B * B > ( A * A ) + (C * C)) | (C * C > (A * A) + (B * B))):
    print("\nE os valores digitados formam um triangulo obtusangulo!")

elif((A * A < (B * B) + (C * C)) | (B * B < ( A * A ) + (C * C)) | (C * C < (A * A) + (B * B))):
    print("\nE os valores digitados formam um triangulo acutangulo")

else:
    print("\n\nOs três números não formam triângulo!")

print("\n\nObrigado por utilizar nosso sistema!")
os.system ("pause")
